### DISCUSSION TITLE
Ibax point calculate to ibax coin

### DISCUSSION THREAD

**Discussion by Memix172** [IBAX-io/go-ibax#906] [Ideas]
How much is calcuate ibax points to ibax coin?

**Comment by Poladxanov** [comment]
i dont know

**Reply by Astalmeta7** [reply]
AİRDROPTAN FAYDALANMAK İÇİN BELLİ BİR PUAN VAR . O PUANI GEÇMEK LAIM VE GEÇEN İLK 1000 KİŞŞİDEN OLMAMIZ GEREKİYOR
